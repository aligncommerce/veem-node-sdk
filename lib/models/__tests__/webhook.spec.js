import Webhook from 'models/webhook'

describe('Webhook Model', () => {
  describe('request', () => {
    it('should match the snapshot', () => {
      expect(Webhook.request.schema).toMatchSnapshot()
    })

    it('should define a valid schema', () => {
      expect(Webhook.request.schema).to.be.validJsonSchema
    })
  })

  describe('response', () => {
    it('should match the snapshot', () => {
      expect(Webhook.response.schema).toMatchSnapshot()
    })

    it('should define a valid schema', () => {
      expect(Webhook.response.schema).to.be.validJsonSchema
    })
  })
})
