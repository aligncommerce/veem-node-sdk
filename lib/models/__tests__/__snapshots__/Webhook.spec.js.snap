// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Webhook Model request should match the snapshot 1`] = `
Object {
  "properties": Object {
    "callbackUrl": Object {
      "type": "string",
    },
    "event": Object {
      "type": "string",
    },
  },
  "title": "WebhookRequest",
  "type": "object",
}
`;

exports[`Webhook Model response should match the snapshot 1`] = `
Object {
  "properties": Object {
    "callbackUrl": Object {
      "type": "string",
    },
    "event": Object {
      "type": "string",
    },
    "id": Object {
      "type": "number",
    },
    "status": Object {
      "type": "string",
    },
  },
  "title": "WebhookResponse",
  "type": "object",
}
`;
